usrsctplib/netinet/sctp_callout.h

```diff
@@ -52,44 +54,30 @@ __FBSDID("$FreeBSD$");
 
 #define SCTP_TICKS_PER_FASTTIMO 20	/* called about every 20ms */
 
-extern userland_mutex_t sctp_os_timerwait_mtx;
```

Comment:
Note for reviewers:
Because `sctp_os_timer_current_changed` was introduced and it is signaled whenever `sctp_os_timer_current` re-assigned it is no longer necessary to have dedicated mutex to handle blocking stop. The combination of `sctp_os_timer_current`, `sctp_os_timer_current_changed` and `&SCTP_BASE_VAR(timer_mtx)` is used to implement blocking stop.